shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/exception/SQLParsingException.java

```diff
@@ -26,8 +26,12 @@
 public final class SQLParsingException extends ShardingSphereSQLException {
     
     private static final long serialVersionUID = -6408790652103666096L;
-    
+
     public SQLParsingException(final String sql) {
         super(XOpenSQLState.SYNTAX_ERROR, 11000, "You have an error in your SQL syntax: %s", sql);
     }
+
+    public SQLParsingException(final String sql, final Exception cause) {
+        super(cause, XOpenSQLState.SYNTAX_ERROR.getValue(), 11000, "You have an error in your SQL syntax: %s, Error message: %s", sql, cause.getMessage());
```

Comment:
Do you think `You have an error in your SQL syntax: %s, error message: %s` is better?

Reply:
Please update the error code document at the same time.

Reply:
> Please update the error code document at the same time.

I am new to this.  Can you tell me where is the error code document?

Reply:
> Do you think `You have an error in your SQL syntax: %s, error message: %s` is better?

Do you mean Error -> error？ 
done :）

Reply:
> > Please update the error code document at the same time.
> 
> I am new to this. Can you tell me where is the error code document?

@lhyundeadsoul You can refer this document——https://shardingsphere.apache.org/document/current/cn/user-manual/error-code/sql-error-code/.

Reply:
> 
update here, Thx:
https://github.com/apache/shardingsphere/pull/20703
https://github.com/apache/shardingsphere/pull/20702